src/abstract/CSBondCore.sol

```diff
@@ -21,13 +22,13 @@ import { IWithdrawalQueue } from "../interfaces/IWithdrawalQueue.sol";
 ///  - burn
 ///
 /// Should be inherited by Module contract, or Module-related contract.
-/// Internal non-view methods should be used in Module contract with additional requirements (if required).
+/// Internal non-view methods should be used in Module contract with additional requirements (if any).
 ///
 /// @author vgorkavenko
-abstract contract CSBondCore {
+abstract contract CSBondCore is ICSBondCore {
     /// @custom:storage-location erc7201:CSAccounting.CSBondCore
     struct CSBondCoreStorage {
-        mapping(uint256 => uint256) bondShares;
+        mapping(uint256 nodeOperatorId => uint256 shares) bondShares;
```

Comment:
it will create getter. but we already have `getBondShares`

Reply:
no problem here. the mapping is in private struct. so the naming for key and value makes no value for the artifacts